### PR TITLE
railway: add page

### LINKED ISSUE #9476: page request: railway
### Command description

Inspired by Heroku cloud computing platform's CLI
[https://railway.app/](https://railway.app/)

### OS

- [ ] Android
- [X] Linux
- [X] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by LiLittleCat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Closes #9476

### CODE REVIEW

File: pages/common/railway.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# railway
+
+> Connect code to Railway project from the command line.
```
Comment (CleanMachine1) [inline pages/common/railway.md:3]:
```suggestion
> Connect code to a Railway project from the command line.
```

File: pages/common/railway.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://railway.app/>.
+
+- Login to your Railway account:
```
Comment (CleanMachine1) [inline pages/common/railway.md:6]:
```suggestion
- Login to a Railway account:
```

File: pages/common/railway.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`railway login`
+
+- Link to an existing Project under your Railway account or team:
```
Comment (CleanMachine1) [inline pages/common/railway.md:10]:
```suggestion
- Link to an existing Project under a Railway account or team:
```

File: pages/common/railway.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run a local command using variables from the active environment:
+
+`railway run {{cmd}}`
+
+- Deploy the linked project directory (if running from a subdirectory, the project root is still deployed):
+
+`railway up`
+
+- Open an interactive shell to a database directly in the CLI:
```
Comment (CleanMachine1) [inline pages/common/railway.md:26]:
```suggestion
- Open an interactive shell to a database:
```

TLDR commands are all in the CLI so redudant.

**Comment by LiLittleCat** [comment]
Thanks for your suggestions @CleanMachine1

**Review by CleanMachine1 — APPROVED** [review]
thanks.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]
